### DISCUSSION TITLE
Select 多选模式时 路由跳转报错

### DISCUSSION THREAD

**Discussion by afra213174647471** [ant-design/ant-design#41729] [Q&A]
不是因为异步报错，只要把mode="multiple"注释掉就不报错，一旦加上就报如下错误
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in Overflow (created by SelectSelector)
    in SelectSelector (created by Selector)
    in div (created by Selector)
    in Selector (created by TriggerWrapper)
    in TriggerWrapper (created by SingleObserver)

**Comment by zombieJ** [comment]
升级一下 React，React 18 里已经不认为这是需要关注的了。

**Reply by afra213174647471** [reply]
> 升级一下 React，React 18 里已经不认为这是需要关注的了。

大佬，react16.14这个版本有解决的办法吗

**Reply by zombieJ** [reply]
可以劫持一下 console 哈~

补一下 React 官方说明：https://github.com/ant-design/ant-design/issues/41572#issuecomment-1491449544
